src/types/modules/react-native-permissions.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-native-permissions/dist/commonjs/permissions';
```

Comment:
Because of this declaration `{PERMISSIONS, RESULTS}` are of type any, is this expected?

Reply:
@blazejkustra, it was not expected. I've removed the declare module and adjusted the code.
Please have a look, thank you!

Reply:
@blazejkustra This was not expected. I have updated the module declaration.
Please have a look, thank you!